src/debian/control

```diff
@@ -1,14 +1,15 @@
 Source: python-fingerprint
 Maintainer: Philipp Meisberger <team@pm-codeworks.de>
 Section: python
-Priority: optional
+Priority: extra
 Standards-Version: 3.9.8
 Build-Depends: debhelper (>= 9),
                dh-python,
                python (>= 2.7),
                python3 (>= 3.2),
                python-setuptools,
-               python3-setuptools
+               python3-setuptools,
```

Comment:
Btw, i checked out the readme, we do not mention these dependencies in the readme, just `git` and `devscripts`.

Reply:
Yes, we maybe should use `sudo mk-build-deps -i -r -t 'apt-get' debian/control`. This would ask user to install missing build dependencies.

Reply:
Yeah, can you include it in the README please?

But why you added the `-r` flag - i think this should be in the responsibility of the user what he want to do with with package files after installation. Also why you specified the `-t` parameter? It the default not fine?

Reply:
Sure: `-t` is not necessary and `-r` is optional. `sudo mk-build-deps -i debian/control`?